### ISSUE TITLE
Possibility to use local icons

### ISSUE THREAD

**Issue by markusglaetzner** [pawelmalak/flame#440]
Deployment details:

    App version: v2.3.1
    Platform: Docker on Debian 12
    Docker image tag: latest

---

I tried to use custom icons that I have copied to the new folder /app/data/icons (all .png from https://github.com/walkxcode/dashboard-icons), so that I can name the icons with using labels in my docker-compose.yml. But it does not work.

With this option I could prepare all docker containers without further actions. Actually I use labels, but have to manualy upload individual icons for every docker container.

Thanks for this nice dashboard and for your great work :-)

**Comment by markusglaetzner** [comment]
It seems to work when I copy all icons directly into the folder "uploads" and to name the icons directly with "flame.icon=docker-icon-name.png". It is not described, but it works.

**Comment by Rumpick** [comment]
I completely agree that the icons section is very-much inadequate. @pawelmalak is there a list of icons that works stock?
